packages/front-end/pages/sdks/[sdkid].tsx

```diff
@@ -400,15 +400,17 @@ export default function SDKConnectionPage() {
           connected={connection.connected}
           canRefresh={canUpdate && !connection.connected}
           refresh={
-            <Button
-              color="link"
-              className="btn-sm"
-              onClick={async () => {
-                await mutate();
-              }}
-            >
-              <BsArrowRepeat /> re-check
-            </Button>
+            connection.proxy.host ? (
```

Comment:
Do we need this change? When the proxy uses SWR to fetch the payload, it should set connection.connected properly, so this is still useful for making sure the proxy's fetch is still working, even if webhooks are disabled

Reply:
I think we still need this. This change hides the "re-check" button. Otherwise we'd be asking `/sdk-connections/${id}/check-proxy` to ping the proxy `/healthcheck` endpoint. But if host is missing, then it's a network error.